docs/updates/2018-12.md

```diff
@@ -0,0 +1,65 @@
+# December 2018
+
+In the final update for this year we would like to highlight a few updates we have made.
+
+## Evaluation Trial License
+
+To better support scalability and load testing evaluations, we have added a trial license without the limitations of the
+community edition (most notably, the limit of 5 concurrent sessions).
+
+More information can be found on the [license](https://core.qlik.com/licensing/#2-evaluation-trial-license) page.
+
+## Data model experience
+
+With the basic scope in place we have started fine tuning our data modeling tool
+[catwalk](https://github.com/qlik-oss/catwalk). Some of the items we've delivered:
+
+### Support modal state in begin selection
+
+When the user begins selecting items in a filterbox, the list is not sorted until the selection mode is ended, i.e. the
+user unselects the filterbox. Once the selection mode has ended, the selections are accepted and the filterbox is
+sorted with the selected items on top.
+
+![screenshot](../images/modal-state.gif)
+
+### Redirect users based on security level
+
+The catwalk tool automatically redirects to http / https, matching the security level of the websocket connecting to
+engine. E.g. a local connection to ws:// will result in catwalk redirecting to <http://catwalk.core.qlik.com>.
+
+Check it out at [catwalk.core.qlik.com](https://catwalk.core.qlik.com), where catwalk is continuously deployed.
+
+## Hardening the corectl tool
+
+After major restructuring of the commands last month, we have continued polishing in order to prepare the tool for the
+removal of the experimental flag. In addition to this, we added the following features:
+
+### Removing objects
+
+Added support for removing applications or objects within an application (connections, dimensions, measures).
+
+```qlik
+corectl remove app theapp
+corectl remove connection connection-id
+```
+
+### Fetching connections
+
+Previously you could only set connections but not list them. This has been fixed in the new release.
+
+```qlik
+corectl get connections
+```
+
+Keep yourself up-to-date by looking for the latest releases in:
+[corectl](https://github.com/qlik-oss/corectl)
+
+## Upcoming
+
+In January 2019, we will continue hardening our data loading tool corectl and will add usability features and
```

Comment:
The word 'hardening' is used a lot in the updates. Are we always hardening? Maybe use some other wording.